src/components/forms/MultipleSelect/MultipleSelect.js

```diff
@@ -1,55 +1,45 @@
-import React from 'react';
+import React, { useState, useRef, useEffect } from 'react';
 import Col from '../../Skeleton/Col';
 import './MultipleSelect.css';
 import PropTypes from 'prop-types';
 
-class MultipleSelect extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      showOptions: false
+const MultipleSelect = props => {
+  const [showOptions, setShowOptions] = useState(false);
+  const wrapperRef = useRef();
+ 
+  useEffect(() => {
+    const handleOutsideClick = event => {
+      if (wrapperRef.current && !wrapperRef.current.contains(event.target) && showOptions) {
+        setShowOptions(false);
+      } 
     }
-    this.wrapperRef = React.createRef();
-    this.toggleOptions = this.toggleOptions.bind(this);
-    this.addParentPropsToChildren = this.addParentPropsToChildren.bind(this);
-    this.handleOutsideClick = this.handleOutsideClick.bind(this);
-  }
+    document.addEventListener('click', handleOutsideClick);
+    return () => document.removeEventListener('click', handleOutsideClick);
+  }, [showOptions]);
 
-  componentDidMount() {
-    document.addEventListener('click', this.handleOutsideClick);
-  }
-  
-  handleOutsideClick(event) {
-    if (this.wrapperRef.current && !this.wrapperRef.current.contains(event.target) && this.state.showOptions) {
-      this.setState({ showOptions: false });
-    } 
-  }
-  toggleOptions () {
-    this.setState({ showOptions: !this.state.showOptions });
+  const toggleOptions = () => {
+    setShowOptions(!showOptions);
   }
 
-  addParentPropsToChildren(option, key) {
-    const { value, onChange, name } = this.props;
+  const addParentPropsToChildren = (option, key) => {
```

Comment:
Ezeket is lehetne destructureolni a propsból `const MultipleSelect = ({ value , onChange ....})`

Reply:
Fixed in [0cd3b29](https://github.com/rkasza/netflix-roulette/pull/5/commits/0cd3b297078b39a413f47a5afd50d0985f364f48)